src/MonoMod.Utils/DMDGenerators/DMDEmit.EmitCallSite.cs

```diff
@@ -462,6 +463,12 @@ void AddOneArgTypeHelperWorker(Type clsArgument, bool lastWasGenericInst)
                         {
                             AddElementType(type);
                         }
+                        // https://github.com/dotnet/runtime/blob/4e0dc086de5952b87e0fdac43690edb5acc4bd6b/src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/SignatureHelper.cs#L449
+                        // btw module is always null
+                        else if (true || tokenCreator/*signatureHelper.module*/ == null)
+                        {
+                            InternalAddRuntimeType(clsArgument);
+                        }
```

Comment:
Is this the case on .NET Framework as well? Can we add a link to referencesource for the impl?